src/overview.md

```diff
@@ -240,6 +240,23 @@ More information:
 * [`webdev` tool](/tools/webdev)
 * [Web deployment tips](/web/deployment)
 
+#### The Dart Runtime {#runtime}
+
+Regardless of how you compile your Dart code, a Dart runtime is needed to
+execute the code. This runtime is responsible for a number of critical tasks:
+
+* Managing memory: Dart uses a managed memory model, where unused memory is
+  reclaimed by a garbage collector (GC). 
+
+* Enforcing the Dart type system: While most type checks are static (compile
+  time), some type checks are dynamic (run time), e.g. the Dart
```

Comment:
Not exactly sure how to restructure this currently to make it sound best, but we may want to consider moving from the "e.g." to an alternative such as "For example," as suggested in [Abbreviations not to use](https://developers.google.com/style/abbreviations?hl=en#dont-use).